bcdaworker/health.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/CMSgov/bcda-app/bcda/health"
+	"github.com/pborman/uuid"
+	"github.com/sirupsen/logrus"
+)
+
+func NewHealthLogger() *HealthLogger {
+	l := HealthLogger{logrus.New()}
+	l.Logger.Formatter = &logrus.JSONFormatter{}
+	filePath := os.Getenv("WORKER_HEALTH_LOG")
+
+	/* #nosec -- 0640 permissions required for Splunk ingestion */
+	file, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0640)
+
+	if err == nil {
+		l.Logger.SetOutput(file)
+	} else {
+		l.Logger.Info("Failed to open worker health log file; using default stderr")
+	}
+
+	return &l
+}
+
+type HealthLogger struct {
+	Logger *logrus.Logger
+}
+
+func (l *HealthLogger) Log() {
+	entry := &HealthLogEntry{Logger: logrus.NewEntry(l.Logger)}
+
+	logFields := logrus.Fields{}
+	logFields["type"] = "health"
+	logFields["id"] = uuid.NewRandom()
```

Comment:
Interesting idea.  Is a UUID for every message a pattern we're following?

Reply:
It was requested in @rnagle's comment on the ticket. Now I'm wondering if we should be including some kind of identifier for the worker. 🤔